src/Api/Controllers/AccountsController.cs

```diff
@@ -1,57 +1,62 @@
-using System.Linq;
-using Bit.Core.Repositories;
-using Bit.Core.Utilities;
+﻿using Bit.Api.Utilities;
 using Bit.Core;
+using Bit.Core.Enums;
+using Bit.Core.Exceptions;
+using Bit.Core.Models.Api;
+using Bit.Core.Models.Api.Request.Accounts;
 using Bit.Core.Models.Business;
-using Bit.Api.Utilities;
+using Bit.Core.Models.Data;
 using Bit.Core.Models.Table;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Core.Utilities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
 using System.Collections.Generic;
-using Bit.Core.Models.Api.Request.Accounts;
-using Bit.Core.Models.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 
 namespace Bit.Api.Controllers
 {
     [Route("accounts")]
     [Authorize("Application")]
     public class AccountsController : Controller
     {
-        private readonly IUserService _userService;
-        private readonly IUserRepository _userRepository;
+        private readonly GlobalSettings _globalSettings;
         private readonly ICipherRepository _cipherRepository;
         private readonly IFolderRepository _folderRepository;
+        private readonly IOrganizationService _organizationService;
         private readonly IOrganizationUserRepository _organizationUserRepository;
         private readonly IPaymentService _paymentService;
-        private readonly GlobalSettings _globalSettings;
+        private readonly IUserRepository _userRepository;
+        private readonly IUserService _userService;
 
         public AccountsController(
-            IUserService userService,
-            IUserRepository userRepository,
+            GlobalSettings globalSettings,
             ICipherRepository cipherRepository,
             IFolderRepository folderRepository,
+            IOrganizationService organizationService,
             IOrganizationUserRepository organizationUserRepository,
             IPaymentService paymentService,
-            GlobalSettings globalSettings)
+            ISsoUserRepository ssoUserRepository,
+            IUserRepository userRepository,
+            IUserService userService)
         {
-            _userService = userService;
-            _userRepository = userRepository;
             _cipherRepository = cipherRepository;
             _folderRepository = folderRepository;
+            _globalSettings = globalSettings;
+            _organizationService = organizationService;
             _organizationUserRepository = organizationUserRepository;
             _paymentService = paymentService;
-            _globalSettings = globalSettings;
+            _userRepository = userRepository;
+            _userService = userService;
         }
 
         [HttpPost("prelogin")]
         [AllowAnonymous]
-        public async Task<PreloginResponseModel> PostPrelogin([FromBody]PreloginRequestModel model)
+        public async Task<PreloginResponseModel> PostPrelogin([FromBody] PreloginRequestModel model)
```

Comment:
Please re-remove these spaces. Not sure why your VSCode is updating the formatting of the entire file, may need to turn that off in your settings.

Reply:
I formatted the whole file to catch anything that might have just been missed in review. Formatting in general has been the bane of my existence on these projects, obviously. Is everyone else using Rider w/the defaults for formatting? I thought I remember hearing VS Code defaults was the style.

Reply:
Yeah, so we're generally using VSCode only for the TypeScript projects, for everything else folks are using Rider or VS for Mac (or Kyle uses Visual Studio Enterprise on Windows), but the defaults for VS for Mac and VS and Rider seem to be a bit more inline vs. the defaults for VSCode in the C# world.

Reply:
Gotcha. I'm pretty attached to my dev environment, which is why this has continued to be an issue (sorry), but once I'm not spending every working second stressing about finishing these SSO tickets I'm going to give all configs some TLC and get some of these redundant formatting issues cleared up. In the meantime I'll run everything through the formatter on VS for Mac and before PR'ing and see if that helps.

Fixed this one.